src/plugins/intel_cpu/src/emitters/jit_load_store_emitters.cpp

```diff
@@ -185,7 +185,10 @@ void jit_load_emitter::load_bytes(const Vmm &vmm, const Xbyak::Reg64 &reg, int o
 
         switch (bytes_to_load) {
             case 0: break;
-            case 1: h->uni_vpinsrb(xmm, xmm, addr(start_bytes), 0); break;
+            case 1: // h->uni_vpinsrb(xmm, xmm, addr(start_bytes), 0); break;
+                h->movzx(Reg32(aux_gpr_idxs[0]), addr(start_bytes));
+                h->uni_vmovq(xmm, Reg64(aux_gpr_idxs[0]));
```

Comment:
Have you checked the same changes for store emitter?
In [Interpolate](https://github.com/openvinotoolkit/openvino/blob/releases/2022/1/src/plugins/intel_cpu/src/nodes/interpolate.cpp#L1443-L1444) there are the same lines for store emitter.

Reply:
The interesting thing is, for store emitter, converting to traditional store_scalar() manner does not have perf improvement, at least I tested on ADL and TGL. Anyway, I replaced with the mov_x() for store scalar, as show no regression as well, in case there are regression on some platforms or code context I have not cover. Thanks!